custom/logistics/api.py

```diff
@@ -234,15 +247,13 @@ def sms_user_sync(self, ilsgateway_smsuser, username_part=None, password=None,
                 user.is_active = bool(ilsgateway_smsuser.is_active)
                 user.user_data = user_dict["user_data"]
                 if "phone_numbers" in user_dict:
```

Comment:
@gcapalbo Look at this line: https://github.com/lwyszomi/commcare-hq/blob/phone-numbers-change/custom/logistics/api.py#L234